src/DBAL/TestStatement.php

```diff
@@ -0,0 +1,206 @@
+<?php declare(strict_types=1);
+
+namespace Fazland\DoctrineExtra\DBAL;
+
+use Doctrine\DBAL\Driver\Statement;
+use Doctrine\DBAL\FetchMode;
+use Doctrine\DBAL\ParameterType;
+
+class TestStatement implements \IteratorAggregate, Statement
```

Comment:
Is necessary this class into the `Fazland\DoctrineExtra\DBAL` namespace?
I imagine that you wrote this for test purposes, am I wrong?

Reply:
For test purposes yes, but re-usable from others libs/projects.